cookbook/extending-website-search.rst

```diff
@@ -0,0 +1,35 @@
+Extending website search
```

Comment:
Not sure about the place should this not be in the `SuluSearchBundle`?

Reply:
Good point, that file does even already exist: https://github.com/sulu/sulu-docs/blob/master/bundles/search.rst